sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -604,11 +610,13 @@ static int sof_audio_probe(struct platform_device *pdev)
 						SOF_RT5682_SSP_CODEC(2);
 	} else {
 		dmic_be_num = 2;
-		hdmi_num = 3;
+		hdmi_num = (sof_rt5682_quirk & SOF_RT5682_NUM_HDMIDEV_MASK) >>
+			 SOF_RT5682_NUM_HDMIDEV_SHIFT;
+		/* default number of HDMI DAI's */
+		if (hdmi_num == 0)
```

Comment:
@sathya-nujella looks good now. just a minor ask. can you change this to if(!hdmi_num)?

Reply:
Sure @ranj063 . I have updated patch set now. Thanks!